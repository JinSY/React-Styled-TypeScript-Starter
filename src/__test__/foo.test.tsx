import React from 'react';
import { shallow } from 'enzyme';

import App from '../app';

test('test', () => {
  expect(true).toBeTruthy();
});

test('snapshot', () => {
  const result = shallow(<App />);

  expect(result).toMatchSnapshot();
})