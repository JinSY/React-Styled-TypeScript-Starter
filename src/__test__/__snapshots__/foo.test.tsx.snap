// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ForwardRef>
        React Styled-Components TypeScript
      </ForwardRef>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": "React Styled-Components TypeScript",
      },
      "ref": null,
      "rendered": "React Styled-Components TypeScript",
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  height: 30px;
  width: 100%;
  background-color: rgb(120,100,123);
  display: flex;
  justify-content: center;
  align-items: center;
  font-family: \\"Montserrat\\";
  font-size: 30px;
  color: white;
  padding: 16px;
",
          ],
        },
        "displayName": "styled.div",
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ForwardRef>
          React Styled-Components TypeScript
        </ForwardRef>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "React Styled-Components TypeScript",
        },
        "ref": null,
        "rendered": "React Styled-Components TypeScript",
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "rules": Array [
              "
  height: 30px;
  width: 100%;
  background-color: rgb(120,100,123);
  display: flex;
  justify-content: center;
  align-items: center;
  font-family: \\"Montserrat\\";
  font-size: 30px;
  color: white;
  padding: 16px;
",
            ],
          },
          "displayName": "styled.div",
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
